Tribler/Core/Modules/restapi/downloads_endpoint.py

```diff
@@ -453,7 +452,8 @@ def get_files_info_json(self, download):
             elif state == "recheck":
                 download.force_recheck()
             elif state == "move_storage":
-                dest_dir = cast_to_unicode_utf8(parameters['dest_dir'])
+                dest_dir = parameters['dest_dir']
+                dest_dir = dest_dir.decode() if isinstance(dest_dir, (bytes, bytearray)) else str(dest_dir)
```

Comment:
Could you try passing `parameters['dest_dir']` as is, without any conversions? After Py3 and AIOHttp all the parameters should be already unicode.

Reply:
Ah, ok. Gotcha. Will try